snmp/tests/test_e2e_core_profiles/test_profile_riverbed_interceptor.py

```diff
@@ -0,0 +1,77 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .. import common
+from ..test_e2e_core_metadata import assert_device_metadata
+from .utils import (
+    assert_common_metrics,
+    assert_extend_generic_if,
+    create_e2e_core_test_config,
+    get_device_ip_from_config,
+)
+
+pytestmark = [pytest.mark.e2e, common.py3_plus_only, common.snmp_integration_only]
+
+
+def test_e2e_profile_riverbed_interceptor(dd_agent_check):
+    config = create_e2e_core_test_config('riverbed-interceptor')
+    aggregator = common.dd_agent_check_wrapper(dd_agent_check, config, rate=True)
+
+    ip_address = get_device_ip_from_config(config)
+    common_tags = [
+        'snmp_profile:riverbed-interceptor',
+        'snmp_host:riverbed-interceptor.device.name',
+        'device_namespace:default',
+        'snmp_device:' + ip_address,
+        'reverbed_interceptor_model:kept zombies Jaded but driving their but',
+        'reverbed_interceptor_serial_number:but zombies quaintly acted but',
+    ] + []
+
+    # --- TEST EXTENDED METRICS ---
+    assert_extend_generic_if(aggregator, common_tags)
+
+    # --- TEST METRICS ---
+    assert_common_metrics(aggregator, common_tags)
+
+    aggregator.assert_metric('snmp.cpu.usage', metric_type=aggregator.GAUGE, tags=common_tags)
```

Comment:
looks like we are not testing the memory metrics.

You need to add the corresponding OID to the .snmprec first: `1.3.6.1.2.1.25.2.3.1.6.1` and `1.3.6.1.2.1.25.2.3.1.5.1`